@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -1,121 +1,148 @@
-}
+  & .navds-combobox__button-toggle-list {
+    color: var(--ax-bg-neutral-moderate);
+  }
 
-.navds-combobox--readonly .navds-combobox__button-toggle-list {
-  color: var(--a-gray-500);
-}
+  & .navds-combobox__wrapper {
+    border-color: var(--ax-border-subtle);
+    overflow: clip;
+  }
 
-.navds-combobox--readonly .navds-text-field__input,
-.navds-combobox--readonly .navds-combobox__input {
-  background-color: var(--a-surface-subtle);
-  border-color: var(--a-border-subtle);
+  & .navds-text-field__input,
+  & .navds-combobox__input {
+    background-color: var(--ax-bg-neutral-moderate);
+  }
 }
 
 .navds-combobox__button-clear svg,
 .navds-combobox__button-toggle-list svg,
 .navds-combobox__list svg {
-  width: var(--__ac-combobox-icon-size);
-  height: var(--__ac-combobox-icon-size);
+  width: var(--__axc-combobox-icon-size);
+  height: var(--__axc-combobox-icon-size);
 }
 
-.navds-combobox__wrapper-inner.navds-text-field__input {
-  position: relative;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  width: 100%;
-  padding-block: calc(var(--__ac-combobox-wrapper-inner-padding) - var(--__ac-combobox-border-width));
-  padding-inline: var(--__ac-combobox-wrapper-inner-padding);
-}
+.navds-combobox__wrapper-inner {
+  &:has(.navds-combobox__input:focus-visible) {
+    outline: 3px solid transparent;
+    outline-offset: 2px;
+  }
 
-.navds-combobox__wrapper-inner > :first-child {
-  flex: 2;
-}
+  &:has(.navds-combobox__input:focus-visible).navds-combobox__wrapper-inner--virtually-unfocused {
+    outline: none;
+  }
 
-.navds-combobox__wrapper-inner > :last-child {
-  display: flex;
-  flex-flow: row nowrap;
-}
+  &.navds-text-field__input {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    padding-block: calc(var(--__axc-combobox-wrapper-inner-padding) - var(--__axc-combobox-border-width));
+    padding-inline: var(--__axc-combobox-wrapper-inner-padding);
+  }
 
-.navds-combobox__wrapper-inner:hover {
-  cursor: text;
-}
+  & > :first-child {
+    flex: 2;
+  }
+
+  & > :last-child {
+    display: flex;
+    flex-flow: row nowrap;
+  }
 
-.navds-combobox--error .navds-text-field__input:not(:hover, :disabled) {
-  border-color: var(--ac-combobox-error-border, var(--a-border-danger));
-  box-shadow: 0 0 0 1px var(--ac-combobox-error-border, var(--a-border-danger));
+  &:hover {
+    cursor: text;
+  }
 }
 
-.navds-combobox--error
-  .navds-text-field__input:not(:hover, :disabled, .navds-combobox__wrapper-inner--virtually-unfocused):focus-within {
-  outline: 2px solid transparent;
-  outline-offset: 2px;
-  box-shadow:
-    0 0 0 1px var(--ac-combobox-error-border, var(--a-border-danger)),
-    var(--a-shadow-focus);
+.navds-combobox--error {
+  .navds-combobox__wrapper {
+    border-radius: var(--ax-border-radius-medium);
+    outline: 3px solid var(--ax-border-danger-strong);
```

Comment:
Error "border" is 2px. We have solved this by setting border-color to danger and adding a 1px box-shadow with danger. This allows creation of a 2px "border" without changing border-width, and allowing regular focus with "outline" to remain --ax-border-focus.
<img width="311" alt="Screenshot 2024-11-07 at 11 26 16" src="https://github.com/user-attachments/assets/f741f7a8-2d53-413a-9c42-7974283c9cd1">
<img width="251" alt="Screenshot 2024-11-07 at 11 26 18" src="https://github.com/user-attachments/assets/c7af39d8-641d-4948-87c1-98cc6b22462e">

Reply:
maching the border color with the outline / box shadow should work for this yeah. We still want to use outline > box-shadow right (in general)?

Reply:
The change from box-shadow -> outline is mainly for focus-handling. For other use-cases (like the error-border) box-shadow is fine i think 👍

Reply:
hmm, men er litt fan av å bare ha "one thing" her 
<img width="224" alt="image" src="https://github.com/user-attachments/assets/43179982-63e2-4620-a26f-a0e5057b304a">
må holde to ting in sync (kan skape en del slike visuelle bugs med 2 borders), og mer CSS å skrive (laste ned, parse) 🤔. Setter man bare fargen til outline på error == ganske KISS, og er vel ikke teknisk sett feil? 🤔 (jeg har også en distaste for box-shadow pga subtle floating point math som er litt forskjellig på tvers av browsere... er kanskje det samme for outline? 😳 )

Reply:
But how would focus be handled then, focus should always be `--ax-border-focus`-color where possible.

Reply:
focus will still be blue, if it has focus you get the style
```
outline-color: var(--ax-border-focus);
```
otherwise it's red (when it has an error). 

This would be fine right? (you can't have _both_ error and focus outlines at the same time in this case, but that's usually fine, since when you're entering a value, it doesn't need to be red to draw our attention anymore 🤔 )

Reply:
> This would be fine right? (you can't have both error and focus outlines at the same time in this case, but that's usually fine, since when you're entering a value, it doesn't need to be red to draw our attention anymore 🤔 )

I think it could matter since change of color when you are in an error-state often indicates that the error is gone. Made a [demo with the two version here](https://play.tailwindcss.com/aWPDcISITP?file=css), should probably sync this with the designers since this will be the case for textfield, textarea, select etc also

Reply:
did some exploratory changes to that demo https://play.tailwindcss.com/ax4OXSQvO6?file=css, here I set negative offsets to outline-offset to have it "fully cover" the border at rest. I made the original border green so you can see the changes, (in your demo the width of the leftmost one would "shrink" as half of it moved away.

Reply:
Might be missing something, but `input[data-v="1"]` uses a `2px` border-width in the new example, something we can't do in code since it will cause shifts in layout. We can take this discussion in a huddle if that's faster 📞

Reply:
Ah, then i'll leave it the way it is (1px border that turns red when error, 1px box shadow red when error) + 2px outline blue with offset 2 due to focus. The same as v2 in the example. 👍